samples/snippets/schema.py

```diff
@@ -515,6 +515,7 @@ def publish_proto_messages(project_id: str, topic_id: str) -> None:
     from google.protobuf.json_format import MessageToJson
     from google.pubsub_v1.types import Encoding
 
+    # TODO(developer): Replace with protocol buffer schema file
```

Comment:
But then they'd have to change the entire sample, wouldn't they? Lines 534-536 assume it is a state object.